core/src/dird/dird.cc

```diff
@@ -334,33 +329,44 @@ int main(int argc, char* argv[])
   my_config->ParseConfig();
```

Comment:
by not dropping privileges before reading the configuration, we produce a whole lot of security issues. We can do it like that for a proof-of-concept, but we cannot put it in production like that.
In fact you can tell the configuration parser to execute scripts and we definitely don't want to run these as root.